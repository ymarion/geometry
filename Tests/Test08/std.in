# Initializing some objects
C c1 0 0 10
R r1 0 0 20 20
L l1 0 0 20 20
# Now, deleting an object
DELETE c1

