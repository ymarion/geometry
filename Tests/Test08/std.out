OK
R r1 0 0 20 20
L l1 0 0 20 20

